doc/nrf/app_dev/device_guides/nrf54l/features.rst

```diff
@@ -42,6 +61,20 @@ In addition, you can run the :ref:`zephyr:bluetooth-samples` that are included f
 
 For available libraries, see :ref:`lib_bluetooth_services` (|NCS|) and :ref:`zephyr:bluetooth_api` (Zephyr).
 
+Bluetooth Mesh
+--------------
+
+Bluetooth Mesh operates on Bluetooth Low Energy (LE), and is implemented according to Bluetooth Mesh Profile Specification v1.0.1 and Bluetooth Mesh Model Specification v1.0.1.
+For the application core, the |NCS| provides several :ref:`bt_mesh_samples`.
+In addition, you can find Bluetooth Mesh samples with :ref:`Bluetooth samples in Zephyr <zephyr:bluetooth-samples>`.
+
+IEEE 802.15.4
+=============
+
+RADIO's implementation of the IEEE 802.15.4 MAC layer frame technology enhances network efficiency and reliability through intelligent decoding of frame control fields, which manage types, addressing, and control flags.
```

Comment:
Who or what is RADIO, and why is the name in all-caps?

Reply:
reworded